src/core/producer/layer.cpp

```diff
@@ -159,6 +189,66 @@ struct layer::impl
             return draw_frame{};
         }
     }
+
+    std::future<std::wstring> call(const std::vector<std::wstring>& params)
```

Comment:
All of the AMCP responses from this are unusual. They say things like `AUTO PLAY NOT ACTIVE` which will be annoying to parse in a library in any kind of generic way.
It would be better to simply return `FRAMES_LEFT TRUE 123`. While less human readable, it will be much easier to parse without needing to consider every route through the code.